### PR TITLE
[10.0][PORT] porting project_timelog from 9.0 to 10.0

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Review by Railia — CHANGES_REQUESTED** [review]
@GabbasovDinar When I start project_timelog_tour then it is failed on 'Click the button of edit' step

**Comment by GabbasovDinar** [comment]
@Railia tests don't work. I created the issue https://github.com/it-projects-llc/misc-addons/issues/531 . I think the issue is actual for all versions of project_timelog module

**Comment by yelizariev** [comment]
```
2018-06-18 15:09:18,488 2376 WARNING 17633-559-519928-base odoo.fields: Field project.task.datetime_stopline: parameter 'select' is no longer supported; use 'index' instead.
2018-06-18 15:09:18,677 2376 INFO 17633-559-519928-base odoo.modules.registry: module project_timelog: creating or updating database tables
2018-06-18 15:09:19,450 2376 INFO 17633-559-519928-base odoo.addons.auth_crypt.models.res_users: Hashing passwords, may be slow for databases with many users...
2018-06-18 15:09:19,825 2376 INFO 17633-559-519928-base odoo.models: Storing computed values of project.timelog fields corrected_duration, duration, project_name, task_name, work_name
2018-06-18 15:09:19,929 2376 INFO 17633-559-519928-base odoo.modules.loading: loading project_timelog/security/ir.model.access.csv
2018-06-18 15:09:20,120 2376 INFO 17633-559-519928-base odoo.modules.loading: loading project_timelog/views/project_timelog_views.xml
2018-06-18 15:09:21,128 2376 INFO 17633-559-519928-base odoo.modules.loading: loading project_timelog/views/res_config_view.xml
2018-06-18 15:09:21,227 2376 WARNING 17633-559-519928-base odoo.addons.base.ir.ir_model: Creating the ir.model.data menu_timelog_config in module base_setup instead of project_timelog.
2018-06-18 15:09:21,235 2376 INFO 17633-559-519928-base odoo.modules.loading: loading project_timelog/views/project_timelog_templates.xml
```

**Comment by GabbasovDinar** [comment]
@yelizariev DONE

File: project_timelog/static/description/index.html [hunk 1]
```diff
@@ -194,9 +194,9 @@ <h2 class="oe_slogan">Notes</h2>
   <div class="oe_row oe_spaced">
     <div class="oe_span8">
       <h2>Need our service?</h2>
-      <p class="oe_mt32">Contact us by <a href="mailto:apps@it-projects.info">email</a> or fill out <a href="https://www.it-projects.info/page/website.contactus " target="_blank">request form</a></p>
+      <p class="oe_mt32">Contact us by <a href="mailto:it@it-projects.info">email</a> or fill out <a href="https://www.it-projects.info/page/website.contactus " target="_blank">request form</a></p>
       <ul>
-        <li><a href="mailto:apps@it-projects.info">apps@it-projects.info <i class="fa fa-envelope-o"></i></a></li>
+        <li><a href="mailto:it@it-projects.info">it@it-projects.info <i class="fa fa-envelope-o"></i></a></li>
```
Comment (yelizariev) [inline project_timelog/static/description/index.html]: revert back to ``apps@``

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Comment by GabbasovDinar** [comment]
@yelizariev DONE

**Review by Railia — APPROVED** [review]

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
The subtask counter stops and the warning signal is played twice. The first is at the unpredictable time (the color does not change), and  the second is when the subtask counter reaches the subtask completion time from the project_timelog setting.

Browser console error when subtask counter reaches completion time:
Uncaught (in promise) DOMException: The play() request was interrupted by a new load request. https://goo.gl/LdLk22

Testing parameters:
Set warning time: 00:01
Set completion time: 00:05
On runbot with updated and explicit modules.

**Review by Ramil-Mukhametzyanov — APPROVED** [review]
It works fine on fast processors with high speed internet connections.

**Comment by KolushovAlexandr** [comment]
superseded by https://github.com/it-projects-llc/misc-addons/pull/772

**Pull request closed without merging**